org.eclipse.mylyn.context.core/META-INF/MANIFEST.MF

```diff
@@ -6,8 +6,11 @@ Bundle-Version: 3.26.0.qualifier
 Bundle-Activator: org.eclipse.mylyn.internal.context.core.ContextCorePlugin
 Bundle-Localization: plugin
 Require-Bundle: org.eclipse.core.runtime,
+ org.eclipse.ui,
  org.eclipse.mylyn.commons.core;bundle-version="[3.8.0,4.0.0)",
- org.eclipse.mylyn.monitor.core;bundle-version="[3.8.0,4.0.0)"
+ org.eclipse.mylyn.monitor.core;bundle-version="[3.8.0,4.0.0)",
+ org.eclipse.mylyn.monitor.ui,
```

Comment:
The UI dependency is not acceptable for "core" bundle

Reply:
> The UI dependency is not acceptable for "core" bundle

Yes, I will change this. Sorry! :worried: I know that, but I made this mistake because it seems I have done too little in the past month with Eclipse Java plugin development.

Reply:
No problem and please continue your effort.
Mylyn currently is in the state where everyone can find an area to improve. 
I'm not sure what to suggest here as the best approach until we decide on the lowest version of Eclipse to support.